wiki/usage.md

```diff
@@ -23,14 +23,14 @@ $ docker build --tag bats:latest .
 $ docker run -it bats:latest --tap /opt/bats/test
 ```
 
-To mount your tests into the container, first built the image as above. Then, for example with `bats`:
+To mount your tests into the container, first build the image as above. Then, for example with `bats`:
 ```bash
 $ docker run -it -v $(pwd):/opt/bats bats:latest /opt/bats/test
 ```
-This has run the `test/` directory from the bats-core repository inside the bats Docker container.
+This runs the `test/` directory from the bats-core repository inside the bats Docker container.
 
+For test suites that are intended to run in isolation from the project (i.e. tests do not depend on project files outside of the test directory), you can mount the test directory by iteself and execute the tests like so:
 
-Or for test files without any dependencies:
 ```bash
 $ docker run -v $(pwd)/test:/test bats /test
```

Comment:
Two nits, one substantial comment.

I'd prefer `-v "$PWD:/opt/bats"` (including double quotes). Though it doesn't matter in terms of performance, it seems folks aren't aware that in Bash, many times you can use `$PWD` where you would commonly use `$(pwd)`, so I like to promote its use.

Should `bats` be `bats:latest`?

Finally, I don't think `-it` is necessary for any of these commands. If you agree, mind not only updating this document, but the `.travis.yml` command?

Reply:
`$PWD` - good point!

`bats` and `bats:latest` are equivalent, but probably more obvious the include the tag. Will update.

`-i` is useful to kill hanging processes (otherwise has to be done via `docker stop` command), and `-t` to simulate a tty (often not used, but most similar to most test runs). Former important to a user, not a build, and latter probably the inverse. Everything's least-surprising to a new Docker use if both are used I think.